File: C++/0892-Surface-of-Area-of-3D-Shapes/soln.cpp
class Solution {
public:
    int surfaceArea(vector<vector<int>>& grid) {
        int N = grid.size();
        int ans = 0;
        for(int i = 0; i < N; ++i) {
            for(int j = 0; j < N; ++j) {
                ans += (grid[i][j] > 0) * 2 + grid[i][j] * 4;
                if (i > 0) {
                    ans -= 2 * (min(grid[i][j], grid[i - 1][j]));
                }
                if (j > 0) {
                    ans -= 2 * (min(grid[i][j], grid[i][j - 1]));
                }
            }
        }
        return ans;
    }
};

File: Python3/0991-Broken-Calculator/soln.py
class Solution:
    def brokenCalc(self, X: 'int', Y: 'int') -> 'int':
        ans = 0
        while X != Y:
            if X >= Y:
                ans += X - Y
                break
            else:
                if Y % 2 == 1:
                    Y += 1
                    ans += 1
                else:
                    Y //= 2
                    ans += 1
        return ans
